HW01/Vladimir_Sereda/vladimir_sereda_hw_01_t_01.rb

```diff
@@ -0,0 +1,18 @@
+puts 'Number of hourse'
+hours = gets.to_i
+
+puts 'Number of minutes'
+minutes = gets.to_i
+
+def angel(hours, minutes)
+  cloc_hand_position = ((hours * 60) + minutes) * 0.5
+  minute_hand_position = minutes * 6
+  angel_between_them = cloc_hand_position - minute_hand_position
```

Comment:
`them` isn't clear name. Would be better to use the whole name
```suggestion
  angel_between_cloc_and_minute_hand = cloc_hand_position - minute_hand_position
```